e2e/components/listing/details/criteria/criterion/criterion.po.js

```diff
@@ -0,0 +1,25 @@
+class CriterionComponent {
+  constructor() {
+    this.elements = {
+      uiFlagOnElement: 'chpl-certification-criteria',
+    };
+  }
+
+  criterionHeader(criteriaNumber, id) {
+    const flag = this.isUiUpgradeFlagOn();
+    if (flag) {
+      return $(`//*[@id="criterion-id-${id}-header"]`).$('h6');
```

Comment:
This is using the xpath format for IDs. It can be done a lot more clearly using this:

```suggestion
      return $(`#criterion-id-${id}-header`).$('h6');
```

Reply:
Fixed this one.  The id for the "old" header in this method has a space in it, so we can't use the "#" notation.